### ISSUE TITLE
screen reader focus issues on iOS

### ISSUE THREAD

**Issue by crentava** [bitwarden/android#1280]
## Describe the Bug

whenever you interact with certain parts of the app, such as the   password entry or password generation instead of moving to the next element, it moves you back to the element you are currently on
## Steps To Reproduce

1. Go to settings>accessibility>voiceover and switch it on
2. Open the bitwarden application
3. click on ‘Add Item’
4. explore the screen, If you have voiceover on you will see what I mean

## Expected Result

you should be able to swipe through all elements in the region of the app you are in
## Actual Result

The cursor ends up jumping back two the element you are currently on
## Screenshots or Videos

I see no way of adding screenshots or recordings but here’s the link to a screen recording I made
https://www.dropbox.com/s/91fecjeiw0g6j7l/RPReplay_Final1613965877.MP4?dl=0
## Environment

iPhone 11
- Operating system: [e.g. iOS 8.1]
iOS 14.4
- Build Version (go to "Settings" → "About" in the app): [e.g. 2.3.0 (2221)]

Version: 2.8.0 (449)
- Is this a Beta release? [Y/N]
n
## Additional Context

if you end up going to the settings>options part of bitwarden, It does the same thing there. First there is a text label describing what the element is and what it does. If you swipe you will see for example (Depending on where you are) a text field, A button, a switch, etc but if you swipe again it takes you back to the text label that you previously read. It shouldn’t be doing this. I’d be glad if this can be fixed
Thank you

**Comment by cscharf** [comment]
thanks @crentava , we have a larger item the team is currently working on in updating Xamarin Forms and accessibility overall which will hopefully alleviate this issue, however we'll keep this open just in case so we can start crossing these things off the list and making the app user experience for all users of all abilities.

**Comment by vvolkgang** [comment]
Issue migrated to https://github.com/bitwarden/mobile/issues/1280
